sdk/src/main/java/io/opentelemetry/sdk/distributedcontext/DistributedContextHttpTextFormat.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.distributedcontext;
+
+import io.opentelemetry.context.propagation.HttpTextFormat;
+import io.opentelemetry.distributedcontext.DistributedContext;
+import io.opentelemetry.distributedcontext.Entry;
+import io.opentelemetry.distributedcontext.EntryKey;
+import io.opentelemetry.distributedcontext.EntryValue;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/** W3C Trace Context implementation of the HttpTextFormat for a DistributedContext. */
+class DistributedContextHttpTextFormat implements HttpTextFormat<DistributedContext> {
+  static final String TRACEPARENT = "traceparent";
+  static final String TRACESTATE = "tracestate";
+  private static final List<String> FIELDS =
+      Collections.unmodifiableList(Arrays.asList(TRACEPARENT, TRACESTATE));
```

Comment:
I think this is the correlation-context from the w3c.

Reply:
Is there something you think should be renamed in here?